docs/CI_CD.md

```
@@ -0,0 +1,227 @@
+### 5. ワークフローを更新
+```yaml
+- uses: cachix/cachix-action@v15
+  with:
+    name: your-project-name  # 作成したキャッシュ名
+    authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+```
+
+## 📊 CI/CDの読み方
+
+### 成功時
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)